examples/tracer-web/index.js

```diff
@@ -1,11 +1,91 @@
 import { ConsoleSpanExporter, SimpleSpanProcessor } from '@opentelemetry/tracing';
 import { WebTracer } from '@opentelemetry/web';
 import { DocumentLoad } from '@opentelemetry/plugin-document-load';
+import { ZoneScopeManager } from '@opentelemetry/scope-zone';
 
 const webTracer = new WebTracer({
   plugins: [
     new DocumentLoad()
   ]
 });
-
 webTracer.addSpanProcessor(new SimpleSpanProcessor(new ConsoleSpanExporter()));
+
+const webTracerWithZone = new WebTracer({
+  scopeManager: new ZoneScopeManager(),
+  plugins: [
+    new DocumentLoad()
+  ]
+});
+webTracerWithZone.addSpanProcessor(new SimpleSpanProcessor(new ConsoleSpanExporter()));
+
+console.log('Current span is window', webTracerWithZone.getCurrentSpan() === window);
+
+const span1 = webTracerWithZone.startSpan('foo1');
```

Comment:
I think this is OK for now, but what would you think about making the example a little more realistic, e.g. something that queries some public API (maybe even pull from `opentelemetry.io` site or GitHub JSON) for data and writes a span for that request after the button is clicked?